src/routes/concepts/derived-values/derived-signals.mdx

```diff
@@ -16,6 +16,7 @@ In the above example, the `double` function relies on the `count` signal to prod
 When the `count` signal is changed, the `double` function will be called again to produce a new value.
 
 Similarly you can create a derived signal that relies on a store value because stores use signals under the hood.
+Learn more about [stores](/concepts/stores).
```

Comment:
```suggestion
Similarly you can create a derived signal that relies on a store value because stores use signals under the hood.
To learn more about how stores work, [you can visit the stores section](/concepts/stores).
```